docs/qa/environments.rst

```diff
@@ -40,11 +35,10 @@ Staging is a test environment used to test the deployment process. It is useful
 for testing a version of Normandy before it is deployed to users, but is not
 guaranteed to be functional or available at all times.
 
-- **Read-only server:** https://normandy.stage.mozaws.net/
-- **Legacy admin:** https://normandy-admin.stage.mozaws.net/ (VPN Required)
-- **Bearer auth admin:** https://normandy-admin-bearer.stage.mozaws.net/ (VPN Required)
+- **Public API server:** https://stage.normandy.nonprod.cloudops.mozgcp.net/
+- **Admin API server:** https://stage-admin.normandy.nonprod.cloudops.mozgcp.net/ (VPN Required)
 
-Staging does not have a API CDN.
+Staging does not have an API CDN.
```

Comment:
Can we change this to "Stage does not go through a CDN for its non-admin URL" or something?

Reply:
I don't understand why this change is different. Can you elaborate on what is wrong here, and why you think your proposed change helps with that problem?

Reply:
You corrected the English but it caused me to notice the sentence and hence me taking the opportunity to mention it could be made easier to read. 

I suspect that the sentence is trying to say is that traffic to the read-only version of the Normandy API does not go through a CDN but instead directly to a web server (Nginx/Django). Is that correct?

Reply:
Right. This was a more important distinction to draw when the document listed a CDN and non-CDN URL for prod. Now it is a minor note to try and explain a discrepancy in the URL structure between stage and prod.